doc/footer.html

```diff
@@ -1,9 +1,9 @@
         </div>
         <div id="footer-container">
             <div id="footer">
-                SFML is licensed under the terms and conditions of the <a href="http://www.sfml-dev.org/license.php">zlib/png license</a>.<br>
+                SFML is licensed under the terms and conditions of the <a href="https://www.sfml-dev.org/license.php">zlib/png license</a>.<br>
                 Copyright &copy; Laurent Gomila &nbsp;::&nbsp;
-                Documentation generated by <a href="http://www.doxygen.org/" title="doxygen website">doxygen</a> &nbsp;::&nbsp;
+                Documentation generated by <a href="https://www.stack.nl/~dimitri/doxygen/" title="doxygen website">doxygen</a> &nbsp;::&nbsp;
```

Comment:
I would keep this. I mean HTTP links aren't really any issue and who knows if that weird URL will move one day, while doxygen.org is more likely to stay.

Reply:
Makes sense I guess.

Reply:
http://www.doxygen.org/ is a redirect to that dutch URL though.

Reply:
I know and? As I said, doxygen.org is more likely to stay than that Dutch URL.

Reply:
Yeah probably right!